### DISCUSSION TITLE
build-setup error

### DISCUSSION THREAD

**Discussion by danielfufu** [ucb-bar/chipyard#1916] [Q&A]
![image](https://github.com/ucb-bar/chipyard/assets/113878158/c9cdc040-9aa3-4025-97a4-29461d604678)
Hi guys!
When I executed ./build-setup.sh riscv-tools, It always failed at step9 (like above)
Could someone help me with this? Thanks!

**Comment by dtkerns** [comment]
try installing the package called libguestfs-tools ... that got me past build-setup error, but I'm still unsuccessful in getting a working installation myself. I've now turned to docker containers.
